ratis-server/src/main/java/org/apache/ratis/server/impl/LeaderStateImpl.java

```diff
@@ -951,9 +951,11 @@ private void yieldLeaderToHigherPriorityPeer() {
     for (LogAppender logAppender : senders.getSenders()) {
       final FollowerInfo followerInfo = logAppender.getFollower();
       final RaftPeerId followerID = followerInfo.getPeer().getId();
-      final RaftPeer follower = conf.getPeer(followerID);
+      final RaftPeer follower = conf.getPeer(followerID, RaftPeerRole.FOLLOWER);
       if (follower == null) {
-        LOG.error("{} the follower {} is not in the conf {}", this, server.getId(), conf);
+        if (conf.getPeer(followerID, RaftPeerRole.LISTENER) == null) {
```

Comment:
Why do you need to add this judgment here? In the election process, it seems that listeners do not need to participate

Reply:
`senders.getSenders()` will return all peers including the listener.

Reply:
Thanks for letting me know, I forgot about this